<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LAS 1.3 Visualiser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="4">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>LiDAR</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Marching Cubes</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="1" column="0" colspan="2">
                  <widget class="QPushButton" name="m_pbCalculate">
                   <property name="text">
                    <string>Polygonise Object</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="2">
                  <widget class="QPushButton" name="m_pbExport">
                   <property name="text">
                    <string>Export Polygon to .obj</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="m_pbClose">
             <property name="text">
              <string>Close App</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0" colspan="3">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Load and Specifications</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="7" column="2">
                  <widget class="QDoubleSpinBox" name="m_sbNoiseLevel">
                   <property name="maximum">
                    <double>200.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>25.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Max</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="2">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>No of Voxels in X axis</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QDoubleSpinBox" name="m_minNorthY"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>North</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="3">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Limits of the area of your interest:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="10" column="2">
                  <widget class="QDoubleSpinBox" name="m_limit">
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>-100.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>1.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>-99.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0" colspan="2">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Length of Voxel</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="2">
                  <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
                 </item>
                 <item row="3" column="1">
                  <widget class="QDoubleSpinBox" name="m_maxNorthY"/>
                 </item>
                 <item row="0" column="0" colspan="3">
                  <widget class="QPushButton" name="m_pbLoadLAS">
                   <property name="text">
                    <string>Load LAS1.3 file</string>
                   </property>
                  </widget>
                 </item>
                 <item row="9" column="2">
                  <widget class="QPushButton" name="m_pbCreateMap">
                   <property name="text">
                    <string>Create</string>
                   </property>
                  </widget>
                 </item>
                 <item row="9" column="0" colspan="2">
                  <widget class="QComboBox" name="m_cbMaps"/>
                 </item>
                 <item row="2" column="2">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Min</string>
                   </property>
                  </widget>
                 </item>
                 <item row="11" column="1" colspan="2">
                  <widget class="QPlainTextEdit" name="m_tHist">
                   <property name="maximumSize">
                    <size>
                     <width>167</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="8" column="0" colspan="3">
                  <widget class="QPushButton" name="m_pbCreate">
                   <property name="text">
                    <string>Create Implicit Object</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QDoubleSpinBox" name="m_minEastX"/>
                 </item>
                 <item row="7" column="0" colspan="2">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Noise Level</string>
                   </property>
                  </widget>
                 </item>
                 <item row="10" column="0" colspan="2">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Object Isolevel</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>East</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="2">
                  <widget class="QSpinBox" name="m_sbNoOfVoxelsInX">
                   <property name="minimum">
                    <number>2</number>
                   </property>
                   <property name="maximum">
                    <number>9999999</number>
                   </property>
                   <property name="value">
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QDoubleSpinBox" name="m_maxEastX"/>
                 </item>
                 <item row="11" column="0">
                  <widget class="QPushButton" name="m_pbSaveHist">
                   <property name="text">
                    <string>Save Hist</string>
                   </property>
                  </widget>
                 </item>
                </layout>
                <zorder>m_pbLoadLAS</zorder>
                <zorder>label_2</zorder>
                <zorder>label_6</zorder>
                <zorder>label_5</zorder>
                <zorder>label_3</zorder>
                <zorder>label_4</zorder>
                <zorder>m_maxNorthY</zorder>
                <zorder>m_maxEastX</zorder>
                <zorder>m_minNorthY</zorder>
                <zorder>m_minEastX</zorder>
                <zorder>m_pbCreate</zorder>
                <zorder>label_7</zorder>
                <zorder>m_sbNoOfVoxelsInX</zorder>
                <zorder>label_8</zorder>
                <zorder>m_limit</zorder>
                <zorder>m_sbNoiseLevel</zorder>
                <zorder>label_9</zorder>
                <zorder>m_tHist</zorder>
                <zorder>m_pbSaveHist</zorder>
                <zorder>label</zorder>
                <zorder>doubleSpinBox</zorder>
                <zorder>m_pbCreateMap</zorder>
                <zorder>m_cbMaps</zorder>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Hyperspectral</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="m_pbLoadHyper">
                <property name="text">
                 <string>Load Hyperspectral images</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="m_sbBand1">
                <property name="maximum">
                 <number>252</number>
                </property>
                <property name="value">
                 <number>145</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Green Band</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Blue Band</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="m_sbBand3">
                <property name="maximum">
                 <number>252</number>
                </property>
                <property name="value">
                 <number>38</number>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Red Band</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="m_sbBand2">
                <property name="maximum">
                 <number>252</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>75</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QPushButton" name="m_updateBands">
                <property name="text">
                 <string>Update Bands</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>Disable Hyperspectral texture</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
